src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -764,6 +764,17 @@ sealed abstract class Stream[+A] extends AbstractSeq[A] with LinearSeq[A] with L
 
   override protected[this] def writeReplace(): AnyRef =
     if(headDefined && tailDefined) new LazyListOps.SerializationProxy[A, Stream](this) else this
+
+  /** Prints elements of this stream one by one, separated by commas. */
+  @deprecated("Use print(stream.force.mkString()) instead")
```

Comment:
I guess the suggestion should rather be to use `print(stream.force.mkString(", "))`?

Reply:
updated. that makes sense.